src/LaunchDarkly.EventSource/Configuration.cs

```diff
@@ -128,9 +121,11 @@ public TimeSpan MaximumDelayRetryDuration
         /// <param name="requestHeaders">Request headers used when connecting to the remote EventSource API.</param>
         /// <param name="lastEventId">The last event identifier.</param>
         /// <param name="logger">The logger used for logging internal messages.</param>
-        /// <param name="closeOnEndOfStream>If true, the EventSource connection will be closed when the end of the stream is reached.</param>
-        /// <exception cref="ArgumentOutOfRangeException">If the delayRetryDuration value is greater than 30 seconds, an ArgumentOutOfRangeException will be thrown.</exception>
-        public Configuration(Uri uri, HttpMessageHandler messageHandler = null, TimeSpan? connectionTimeOut = null, TimeSpan? delayRetryDuration = null, TimeSpan? readTimeout = null, IDictionary<string, string> requestHeaders = null, string lastEventId = null, ILogger logger = null, bool closeOnEndOfStream = false)
+        /// <exception cref="ArgumentNullException">Throws ArgumentNullException if the uri parameter is null.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">Throws ArgumentOutOfRangeException if the connectionTimeOut value is less than zero.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">Throws ArgumentOutOfRangeException if the delayRetryDuration value is greater than 30 seconds.</exception>
```

Comment:
It might be better to combine  all the cases for a single exception into a single entry: https://stackoverflow.com/questions/6434499/xml-comments-how-to-comment-multiple-reasons-for-an-exception

Reply:
Yep, makes sense. I'll update this.